### PR TITLE
[fix][doc] fix doc and template error

### PR DESCRIPTION
**Pull request by labuladong**
### Modifications

Fix some errors in doc and template.

### Verifying this change

- [x] Make sure that the change passes the CI checks.

This change is a trivial rework / code cleanup without any test coverage.

### Documentation

- [x] `doc` 
- [ ] `doc-required` 
- [ ] `doc-not-needed` 
- [ ] `doc-complete`

### CODE REVIEW

**Comment by labuladong** [comment]
cc @tisonkun @Anonymitaet @momo-jun

File: site2/docs/functions-overview.md [hunk 1]
```diff
@@ -26,9 +26,9 @@ Each time a function receives a message, it completes the following consume-appl
 1. Consumes the message from one or more **input topics**. 
 2. Applies the customized (user-supplied) processing logic to the message.
 3. Publishes the output of the message, including:
-    a) writes output messages to an **output topic** in Pulsar
-    b) writes logs to a **log topic** (if it is configured)for debugging
-    c) writes [state](functions-develop-state.md) updates to BookKeeper (if it is configured) 
+    - writes output messages to an **output topic** in Pulsar
+    - writes logs to a **log topic** (if it is configured)for debugging
+    - writes [state](functions-develop-state.md) updates to BookKeeper (if it is configured)
```
Comment (tisonkun) [inline site2/docs/functions-overview.md:31]: I think the original list is an ordered list?

**Review by tisonkun — COMMENTED** [review]
Thank you! One comment inline.

File: site2/docs/functions-overview.md (hunk 1 above)
Comment (labuladong) [reply site2/docs/functions-overview.md:31]: This abc list did not display:
image

So change it to an unordered list.

Comment (tisonkun) [reply site2/docs/functions-overview.md:31]: Yes. I mean if the original purpose is an ordered list, you may use:

```suggestion
    1. writes output messages to an **output topic** in Pulsar
    2. writes logs to a **log topic** (if it is configured)for debugging
    3. writes [state](functions-develop-state.md) updates to BookKeeper (if it is configured) 
```

Comment (labuladong) [reply site2/docs/functions-overview.md:31]: Make sense

**Review by tisonkun — APPROVED** [review]

**Review by momo-jun — APPROVED** [review]

**Pull request merged**